docs/guide/extending-default-theme.md

```diff
@@ -26,15 +26,109 @@ import './custom.css'
 export default DefaultTheme
 ```
 
+The default theme CSS is customizable by overriding root level CSS variables. This example shows how to change the main brand color from VitePress green to a shade of purple:
+
 ```css
 /* .vitepress/theme/custom.css */
 :root {
+  /* The main brand color:
+     Typically used for Hero text, links, "brand" buttons, 
+     search result boxes, and other highlights */
   --vp-c-brand: #646cff;
-  --vp-c-brand-light: #747bff;
+
+  /* Lighter variation of brand color:
+     Typicaly used for the border of brand-colored buttons */
+  --vp-c-brand-lighter: #9da2fd;
+
+  /* Darker variation of brand color:
+     Typically used for brand button hover state */
+  --vp-c-brand-dark: #373ec3;
+  
+  /* Even darker variation of brand color
+     Typically used for brand button active state */
+  --vp-c-brand-darker: #3030c0;
 }
 ```
 
-See [default theme CSS variables](https://github.com/vuejs/vitepress/blob/main/src/client/theme-default/styles/vars.css) that can be overridden.
+See [`vars.css`](https://github.com/vuejs/vitepress/blob/main/src/client/theme-default/styles/vars.css) in the default theme for other variables that can be overridden.
+
+You can also apply any other CSS style rules here. While using CSS variables is good practice to help reduce repetition, plain CSS is also fine.
+
+Just remember to choose colors that work well in both dark and light modes, unless you disable the dark mode toggle or specify unique colors for both light and dark modes. See [Dark Mode](#dark-mode) below for more.
+
+## Using CSS Pre-Processors
+
+If you'd like to use SASS, LESS, or Stylus within your theme, first make sure the relevant pre-processor is installed by running the applicable command:
+
```

Comment:
What's the benefit of the `@latest` suffix? It's not something I've really used before. Doesn't `npm install` typically fetch the latest stable version by default? Or does `@latest` grab alpha/beta versions?

Reply:
Just circling back to your comment from last year. Is there any benefit in specifying `@latest` when running `npm install`? 

According to the [npm documentation](https://docs.npmjs.com/cli/v10/commands/npm-dist-tag#description):

> By default, `npm install <pkg>` (without any `@<version>` or `@<tag>` specifier) installs the latest tag.

Is this not necessarily the case with other package managers?

Or is `@latest` a way of overriding whatever version might already be present in your `package.json`?

I couldn't see any places where `@latest` is used in the current VitePress docs, but if that is the preferred syntax we can go with that for consistency. But it would just be good to understand the intention behind that.